be/src/runtime/mem_tracker.cpp

```diff
@@ -60,6 +60,21 @@ MemTracker* MemTracker::get_raw_process_tracker() {
     return raw_process_tracker;
 }
 
+// Track memory for all brpc server responses.
+static std::shared_ptr<MemTracker> brpc_server_tracker;
+static GoogleOnceType brpc_server_tracker_once = GOOGLE_ONCE_INIT;
+
+void MemTracker::create_brpc_server_tracker() {
+    brpc_server_tracker.reset(new MemTracker(-1, "Brpc", get_process_tracker(), MemTrackerLevel::OVERVIEW, nullptr));
```

Comment:
I think we should use `MemTracker::create_tracker`? And no need to call `add_child_tracker` and `init` manually.

Reply:
done, you are right